### PR TITLE
added first draft of updates to development.md

### PR DESCRIPTION
**Pull request by tarockey**
- Updated development.md to include steps required to deploy the ARM templates, and then to use the Dev Jump Box to deploy SDS in the AKS cluster behind an Azure Firewall.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -120,3 +179,7 @@ It can be useful to execute the samples rapidly. Scripts are provided to help yo
 
 * `scripts/sample-data.sh` - Execute the basic YAML files
 * `scripts/demo-catdetection.sh` - Run the cat detection demo (see [README](/samples/catdetection/README.md) for setup instructions)
+
```
Comment (tessmichi) [inline docs/development.md:182]: nit: extra newlines

**Review by tessmichi — APPROVED** [review]
love these docs :)

File: docs/development.md [hunk 2]
```diff
@@ -28,6 +28,41 @@ To fully develop and debug SDS, you'll want to have the following installed on y
...
+- Azure Firewall
+- Private Link configurations and Virtual Networks
+- Azure Storage accounts
+- Azure Container Registry
+
+To deploy the ARM template, first create an SSH key on the machine you intend to execute the ARM template from - it should be named id_rsa (the default name when using [ssh keygen](https://www.tutorialspoint.com/unix_commands/ssh-keygen.htm)).
+
+#### Using local templates (optional)
+
+>If you are developing changes in the ARM templates and wish to use them directly, you first need to make them available over the internet (due to limitations in the Azure CLI). The easiest way to do this is using the [ngrok](https://ngrok.com/download) tool. Once you have installed the ngrok tool, you can expose the ARM template folder by running the command:
```
Comment (noelbundick) [inline docs/development.md:48]: Can we update this to point to using the tools that are baked into the repo? We have an npm script to auto-launch ngrok with the correct settings, and then `deploy.sh` will automatically introspect the running ngrok instance (as long as `--assets` isn't explicitly set) to get the URL & auto-fill the assets URI. Deployment from a fresh clone is:

```shell
# install dependencies
npm install

# Start a local tunnel
npm run dev:azure

# deploy templates
./deploy/arm/deploy.sh -g <resource_group> --dev
```

File: docs/development.md [hunk 3]
```diff
@@ -82,15 +117,39 @@ sudo mv ./kind /usr/local/bin/kind
...
 * `worker.queueEndpoint`: Point to a queue in your Azure Storage Account
 * `identities.*`: Enter values for your own Service Principal(s). The `worker` identity will need the `Storage Queue Data Message Processor` (or higher) role on your queue
 
+>NOTE: If using the ARM template, managed identities have been created for you, and you can instead use the format found in `deploy/helm/values.yaml`, filling in the appropriate client IDs matching the pre-created identities.
+
 ### Install SDS
 
+If this is your first time using the K8s cluster, be sure that you have installed the AAD Identity Binding on the AKS cluster within the `kube-system` namespace.
+```shell
+kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment-rbac.yaml -n kube-system
```
Comment (noelbundick) [inline docs/development.md:149]: We should probably choose a known-good version versus the master branch

**Review by noelbundick — APPROVED** [review]
Approved w/ suggestions

File: docs/development.md (hunk 3 above)
Comment (tarockey) [reply docs/development.md]: Updated to use v1.6.0 release

File: docs/development.md (hunk 2 above)
Comment (tarockey) [reply docs/development.md]: Updated to reflect npm tooling

File: docs/development.md (hunk 1 above)
Comment (tarockey) [reply docs/development.md]: fixed

**Pull request merged**